packages/bot-skeleton/src/scratch/hooks/trashcan.js

```diff
@@ -1,22 +1,23 @@
-const trashcan_margin = 16;
-const core_footer_height = 36;
+export const initTrashCan = (margin = 16, height = 36) => {
+    /**
+     * Distance between trashcanand bottom edge of workspace.
```

Comment:
please quickly add a space 
```suggestion
     * Distance between trashcan and bottom edge of workspace.
```

Reply:
done ✅